### PR TITLE
Remove rawInputDataSize and rawInputPositions from stats

### PR DESCRIPTION
**Pull request by raunaqmorarka**

### CODE REVIEW

File: presto-benchmark/src/main/java/io/prestosql/benchmark/AbstractOperatorBenchmark.java [hunk 1]
```diff
@@ -306,8 +306,8 @@ protected final OperatorFactory createHashProjectOperator(int operatorId, PlanNo
         CpuDuration executionTime = cpuTimer.elapsedTime();
 
         TaskStats taskStats = taskContext.getTaskStats();
-        long inputRows = taskStats.getRawInputPositions();
-        long inputBytes = taskStats.getRawInputDataSize().toBytes();
+        long inputRows = taskStats.getPhysicalInputPositions() + taskStats.getInternalNetworkInputPositions();
```
Comment (sopel39) [inline presto-benchmark/src/main/java/io/prestosql/benchmark/AbstractOperatorBenchmark.java:309]: you can use `io.prestosql.operator.TaskStats#getProcessedInputPositions`

File: presto-main/src/main/java/io/prestosql/connector/system/TaskSystemTable.java [hunk 2]
```diff
@@ -61,16 +61,16 @@
...
-            .column("raw_input_rows", BIGINT)
-
             .column("processed_input_bytes", BIGINT)
             .column("processed_input_rows", BIGINT)
 
             .column("output_bytes", BIGINT)
             .column("output_rows", BIGINT)
 
             .column("physical_input_bytes", BIGINT)
+            .column("physical_input_rows", BIGINT)
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/connector/system/TaskSystemTable.java:71]: make this separate commit

File: presto-benchmark/src/main/java/io/prestosql/benchmark/AbstractOperatorBenchmark.java [hunk 3]
```diff
@@ -306,8 +306,8 @@ protected final OperatorFactory createHashProjectOperator(int operatorId, PlanNo
...
+        long inputBytes = taskStats.getPhysicalInputDataSize().toBytes() + taskStats.getPhysicalInputDataSize().toBytes();
```
Comment (sopel39) [inline presto-benchmark/src/main/java/io/prestosql/benchmark/AbstractOperatorBenchmark.java:310]: this is adding `getPhysicalInputDataSize` twice.

File: presto-main/src/main/java/io/prestosql/event/SplitMonitor.java [hunk 4]
```diff
@@ -88,9 +88,9 @@ private void splitCompletedEvent(TaskId taskId, DriverStats driverStats, @Nullab
                                     ofMillis(driverStats.getTotalCpuTime().toMillis()),
                                     elapsedTime,
                                     queuedTime,
-                                    ofMillis(driverStats.getRawInputReadTime().toMillis()),
-                                    driverStats.getRawInputPositions(),
-                                    driverStats.getRawInputDataSize().toBytes(),
+                                    ofMillis(driverStats.getPhysicalInputReadTime().toMillis()),
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/event/SplitMonitor.java:91]: There seem to be a bug, see: ```io.prestosql.operator.DriverContext#getDriverStats```:
```
            physicalInputReadTime = inputOperator.getAddInputWall();
            ...
            internalNetworkInputReadTime = inputOperator.getAddInputWall();
```
we shouldn't be adding two of them. Either:
1. there should be two separate fields in `DriverContext` for reporting physical and internal network read times
2. We should remove `internalNetworkInputReadTime` altogether (it's not set by anyone, so it's always 0).

I think we should do 2)

File: presto-main/src/main/java/io/prestosql/server/BasicQueryStats.java [hunk 5]
```diff
@@ -50,9 +50,11 @@
     private final int runningDrivers;
     private final int completedDrivers;
 
-    private final DataSize rawInputDataSize;
-    private final long rawInputPositions;
     private final DataSize physicalInputDataSize;
+    private final long physicalInputPositions;
+
+    private final DataSize internalNetworkInputDataSize;
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/server/BasicQueryStats.java:56]: make this separate commit

File: presto-main/src/main/java/io/prestosql/execution/StageStateMachine.java [hunk 6]
```diff
@@ -318,11 +314,6 @@ public BasicStageStats getBasicStageStats(Supplier<Iterable<TaskInfo>> taskInfos
 
             internalNetworkInputDataSize += taskStats.getInternalNetworkInputDataSize().toBytes();
             internalNetworkInputPositions += taskStats.getInternalNetworkInputPositions();
-
-            if (fragment.getPartitionedSourceNodes().stream().anyMatch(TableScanNode.class::isInstance)) {
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/execution/StageStateMachine.java:322]: that was a bad hack....

File: presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java [hunk 7]
```diff
@@ -536,9 +535,6 @@ private QueryStats getQueryStats(Optional<StageInfo> rootStage)
 
             PlanFragment plan = stageInfo.getPlan();
             if (plan != null && plan.getPartitionedSourceNodes().stream().anyMatch(TableScanNode.class::isInstance)) {
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java:537]: We probably should think about removing this if too, but not as part of this PR

File: presto-main/src/main/java/io/prestosql/server/QueryProgressStats.java [hunk 8]
```diff
@@ -68,8 +68,8 @@ public static QueryProgressStats createQueryProgressStats(BasicQueryStats queryS
                 queryStats.getTotalScheduledTime().toMillis(),
                 queryStats.getUserMemoryReservation().toBytes(),
                 queryStats.getPeakUserMemoryReservation().toBytes(),
-                queryStats.getRawInputPositions(),
-                queryStats.getRawInputDataSize().toBytes(),
+                queryStats.getPhysicalInputPositions() + queryStats.getInternalNetworkInputPositions(),
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/server/QueryProgressStats.java:71]: I don't think `queryStats.getPhysicalInputPositions() + queryStats.getInternalNetworkInputPositions()` is equivalent to `rawInputPositions`. `QueryStateMachine` has:
```
            if (plan != null && plan.getPartitionedSourceNodes().stream().anyMatch(TableScanNode.class::isInstance)) {
                rawInputDataSize += stageStats.getRawInputDataSize().toBytes();
                rawInputPositions += stageStats.getRawInputPositions();

                processedInputDataSize += stageStats.getProcessedInputDataSize().toBytes();
                processedInputPositions += stageStats.getProcessedInputPositions();
            }
```
Addition here is conditional. It seems that `processedInputPositions` is equivalent to `rawInputPositions`. However, there is not equivalence for `rawInputDataSize`.

File: presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java (hunk 7 above)
Comment (sopel39) [reply presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java:537]: In fact, we probably should change semantics of `rawInputDataSize`, `rawInputPositions`, `processedInputDataSize` and `processedInputPositions` before we remove `rawInputDataSize` and `rawInputPositions` altogether. This is because there is no equivalence for `rawInputDataSize` once we remove it and some listeners might rely on it (e.g `SplitCompletedEvent`)

File: presto-main/src/main/java/io/prestosql/server/protocol/Query.java [hunk 9]
```diff
@@ -712,8 +712,8 @@ private static StatementStats toStatementStats(QueryInfo queryInfo)
                 .setWallTimeMillis(queryStats.getTotalScheduledTime().toMillis())
                 .setQueuedTimeMillis(queryStats.getQueuedTime().toMillis())
                 .setElapsedTimeMillis(queryStats.getElapsedTime().toMillis())
-                .setProcessedRows(queryStats.getRawInputPositions())
-                .setProcessedBytes(queryStats.getRawInputDataSize().toBytes())
+                .setProcessedRows(queryStats.getPhysicalInputPositions() + queryStats.getInternalNetworkInputPositions())
```
Comment (sopel39) [inline presto-main/src/main/java/io/prestosql/server/protocol/Query.java:715]: should we just remove it from here?

**Comment by colebow** [comment]
:wave: @raunaqmorarka - this PR is inactive and doesn't seem to be under development. If you'd like to continue work on this at any point in the future, feel free to re-open.
